apps/masterbots.ai/components/routes/chat/chat.tsx

```
@@ -66,87 +64,87 @@ export function Chat({
+        element = sectionRef.current
       } else {
-        element = containerRef.current;
+        element = containerRef.current
       }
-      debounceScrollToBottom(element);
+      debounceScrollToBottom(element)
     }
-  };
+  }
 
   useEffect(() => {
```

Biome (1.9.4)

[error] This hook does not specify all of its dependencies: activeThread

This dependency is not specified in the hook dependency list.

Either include it or remove the dependency array

(lint/correctness/useExhaustiveDependencies)